.github/workflows/release.yaml

```diff
@@ -9,12 +9,16 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Extract Release Notes
+      uses: ffurrer2/extract-release-notes@v1.16.0
```

Comment:
No way! Are we going to have proper release notes now? :heart_eyes:

Reply:
Haha yes that's the idea :smile: We may have also used the "auto-generated" release notes of Github here, but I like to have them look exactly like they look in `CHANGELOG.md`